src/components/BuildConfigsList/BuildConfigsList.tsx

```diff
@@ -0,0 +1,249 @@
+                  </ProtectedComponent>
+                )}
+              </Tr>
+            </Thead>
+            <Tbody>
+              {serviceContainerBuildConfigs.data?.content.map(
+                (buildConfig: BuildConfigurationWithLatestBuild, rowIndex: number) => (
+                  <Tr key={rowIndex}>
+                    {columns.includes(buildConfigEntityAttributes.name.id) && (
+                      <Td>
+                        <BuildConfigLink id={buildConfig.id}>{buildConfig.name}</BuildConfigLink>
+                      </Td>
+                    )}
+                    {columns.includes(buildConfigEntityAttributes.description.id) && <Td>{buildConfig.description}</Td>}
+                    {columns.includes(buildConfigEntityAttributes.buildType.id) && (
+                      <Td>
+                        <BuildConfigBuildTypeLabelMapper buildType={buildConfig.buildType} />
+                      </Td>
+                    )}
+                    {columns.includes(buildConfigEntityAttributes.project.id) && (
+                      <Td>
+                        {buildConfig.project && (
+                          <ProjectLink id={buildConfig.project?.id}>{buildConfig.project?.name}</ProjectLink>
+                        )}
+                      </Td>
+                    )}
+                    {(columns.includes(buildConfigEntityAttributes.creationTime.id) ||
+                      columns.includes(buildConfigEntityAttributes.modificationTime.id)) && (
+                      <Td>
+                        <DescriptionList className="gap-0" isHorizontal isCompact>
+                          {columns.includes(buildConfigEntityAttributes.creationTime.id) && (
+                            <DescriptionListGroup>
+                              <DescriptionListTerm>{buildConfigEntityAttributes.creationTime.title}</DescriptionListTerm>
+                              <DescriptionListDescription>
+                                {buildConfig.creationTime && createDateTime({ date: buildConfig.creationTime }).custom}
+                                {buildConfig.creationUser && (
+                                  <span>
+                                    {' '}
+                                    by <b>{buildConfig.creationUser?.username}</b>
+                                  </span>
+                                )}
+                              </DescriptionListDescription>
+                            </DescriptionListGroup>
+                          )}
+                          {columns.includes(buildConfigEntityAttributes.modificationTime.id) && (
+                            <DescriptionListGroup>
+                              <DescriptionListTerm>{buildConfigEntityAttributes.modificationTime.title}</DescriptionListTerm>
+                              <DescriptionListDescription>
+                                {buildConfig.modificationTime && createDateTime({ date: buildConfig.modificationTime }).custom}
```

Comment:
try to use shorter date format in certain conditions with the `includeDateInCustom` using the same logic as that of BuildsList times column